### DISCUSSION TITLE
Getting unwanted trait in metadata

### DISCUSSION THREAD

**Discussion by kayes0001** [nftchef/art-engine#303] [Help + Q&A]
getting extra trait like this in metadata if traits are more than 5...any solutions?

![Capture](https://user-images.githubusercontent.com/60549087/211158959-2fbb96ca-2174-42c4-bfb8-7c8460f030fe.JPG)

I'm novice at coding.

**Comment by nftchef** [comment]
this is almost always because there is a hidden file in the layers folder called `Desktop.ini`
Or, another file that does not have a `#` weight.
 if you delete that or any unintentional unweighted files , it should go away
